addon/mixins/in-viewport.js

```diff
@@ -64,164 +65,164 @@ export default Mixin.create({
   },
 
   _startListening() {
-    this._setInitialViewport(window);
-    this._addObserverIfNotSpying();
-    this._bindScrollDirectionListener(window, get(this, 'viewportScrollSensitivity'));
-
-    if (!get(this, 'viewportUseRAF')) {
-      get(this, 'viewportListeners').forEach((listener) => {
-        const { context, event } = listener;
-        this._bindListeners(context, event);
-      });
-    }
-  },
-
-  _addObserverIfNotSpying() {
-    if (!get(this, 'viewportSpy')) {
-      this.addObserver('viewportEntered', this, this._unbindIfEntered);
-    }
+    this._setInitialViewport();
+    // this._addObserverIfNotSpying();
+    // this._bindScrollDirectionListener(window, get(this, 'viewportScrollSensitivity'));
```

Comment:
To allow for backwards compat (if that is the goal), we would probably want to keep this around for `didScroll`